modules/vye/lib/dgib/configuration.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Vye
+  module DGIB
+    class Configuration < Common::Client::Configuration::REST
+      def connection
+        @conn ||= Faraday.new(base_path, headers: base_request_headers, request: request_options) do |faraday|
+          faraday.use :breakers
+          faraday.ssl[:ca_file] = '/srv/vets-api/secret/dgib_ica11_rca2.key'
```

Comment:
Please reference Settings

Reply:
Regarding the first recommendation above, iirc the policy IS tied to the user_info. Accordingly I refactored the controller to only use access? as per your recommendation. I also refactored the user_info_policy.rb to remove the four it had and replace them with access?

Regarding the second change. I'm not following here. I took out the commented line, but the green lines you have listed are already there. Not sure what the ask is here.

Reply:
'/srv/vets-api/secret/dgib_ica11_rca2.key' is not a valid path and will be null. Please reference this [cert](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/6c1a4b1cd0a05b53162a87fd4ceb53891a3ae183/apps/vets-api/dev/values.yaml#L2676) aka `Settings.dgi.vye.public_ica11_rca2_key_path`